solutions/2021/day_06/README.md

```diff
@@ -75,8 +75,13 @@ reg['a'] += 1
 # => KeyError: 'a'
 
 # Instead, we need to write:
+if 'a' in reg:
+    reg['a'] += 1
+else:
+    reg['a'] = 1
 
-reg['a'] = reg.get('a', int()) + 1
+# or, as a one-liner:
+# reg['a'] = reg.get('a', int()) + 1
```

Comment:
the reason i wrote it this way is because that is actually how it gets implemented by the collections module itself: https://github.com/python/cpython/blob/d460c8ec52716a37080d31fdc0f673edcc98bee8/Modules/_collectionsmodule.c#L1951-L1973

Reply:
totally fair! I was purposefully keeping the example simple and verbose for learning purposes (I think adding `.get()` and `int()` are probably out of scope for day 6), but I appreciate the suggestion.